app/src/main/java/com/example/danik/google_books_application/Constants.java

```diff
@@ -0,0 +1,8 @@
+package com.example.danik.google_books_application;
+
+public class Constants {
+    public final static String preference = "Prefs";
+    public final static String preferenceRecord = "Record";
+    public final static String favorites = "Favorites";
+    public final static String ARG_TITLE = "Item";
```

Comment:
According to Java code convention, constants should have names with all letters capital